Makefile

```diff
@@ -31,6 +31,7 @@ create/mamba:  ## create conda environment "webmon" with file conda_environment.
 
 docker/pruneall: ## stop and force remove all containers, images and volumes
 	docker system prune --force --all --volumes
+	docker system prune --force --all --volumes
```

Comment:
Why is it done twice?

Reply:
bummer, looks like I didn't resolve all the conflicts. I'll remove the extra line